open-machine-learning-jupyter-book/ml-advanced/ensemble-learning/getting-started-with-ensemble-learning.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>The last sentence can be pointed out as emsemble learning, leading to the video.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ocademy-ai/machine-learning/pull/757/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
done

Reply:
<p>Also, it's best to split into multiple code blocks to make the structure logic clearer.</p>

Reply:
<p>Perhaps “Ensembles” would be better represented by secondary headings, rather than bolded?</p>

Reply:
done cant use the secondary headings cuz its the introduction part

Reply:
Secondary headings mean that they are automatically recognized as secondary headings when preceded by ##, as in ## Acknowledgments